testing/python_lib/unit_base.py

```diff
@@ -70,12 +76,12 @@ def __init__(self, *args, **kwargs):
         self._forchestrator = None
 
     def _setup_env(self):
-        assert self._temp_dir
-        os.environ['FORCH_CONFIG_DIR'] = os.path.join(self._temp_dir, 'forch')
-        os.environ['FORCH_CONFIG_FILE'] = os.path.basename(self._temp_forch_config_file)
-        os.environ['FAUCET_CONFIG_DIR'] = os.path.join(self._temp_dir, 'faucet')
-        os.environ['FAUCET_CONFIG_FILE'] = os.path.basename(self._temp_behavioral_config_file)
-        os.environ['FAUCET_EVENT_SOCK'] = self._temp_socket_file
+        assert self._base_dir
```

Comment:
why assert _base_dir here?  It's not used in this function...

Reply:
Right, it's not used below in that function. Removed. PTAL..